Makefile

```diff
@@ -477,18 +461,10 @@ install-manpages:
 	cp man/* /usr/local/share/man/man1/
 	mandb
 
-# Strip "tabs assets" errors from the dummy target, but fail on target failure.
-check-docs:
-	$(QUIET)($(MAKE) -C Documentation/ dummy SPHINXOPTS="$(SPHINXOPTS)" 2>&1 && touch $@.ok) \
-		| grep -v "tabs assets"
-	@rm $@.ok 2>/dev/null
-
 postcheck: build
-	@$(ECHO_CHECK) contrib/scripts/check-cmdref.sh
-	$(QUIET) MAKE=$(MAKE) contrib/scripts/check-cmdref.sh
+	$(QUIET)$(MAKE) -C Documentation update-cmdref check-cmdref
```

Comment:
(not related to this specific line, but in this area)

If I currently use `make check-docs` or `make test-docs` for simple docs validation, what do you recommend I do instead? `make -C Documentation`? `make render-docs`? (I guess the latter is generally more popular anyway)

Reply:
Yeah `make render-docs` will run the general check logic, but it also creates the Apache container, which is what it used to do also. I think `make -C Documentation` (alias for `make -C Documentation html`) will do the the checks and just write the HTML files, I avoided promoting too many new targets to the top-level makefile for the time being.

I just double-checked, and `make -C Documentation html` certainly covers what `make check-docs` as well as `make test-docs` did.